set_default.cpp

```diff
@@ -68,13 +68,28 @@ void set_default_input_values(void) {
   g_resume    = _default_resume;
   g_subtract  = _default_subtract;
   g_source_location = _default_source_location;
-  int count=snprintf(spinprojectionfname_prefix,200, "%s",  _default_spinprojectionfname_prefix);
+  int count=snprintf(g_LoopExtract_InPath,200, "%s",  _default_LoopExtract_InPath);
```

Comment:
I think the path and filename may need to be longer. It's easily possible to generate a filename with well over 200 characters with the deep directory structures that we have to use...